mcache/tests/test_check.py

```diff
@@ -208,10 +267,22 @@ def test_service_ok(check, instance, aggregator, memcached):
     """
     Service is up
     """
-    tags = ["host:{}".format(HOST), "port:{}".format(PORT), "foo:bar"]
+    tags = ["host:{}".format(common.HOST), "port:{}".format(common.PORT), "foo:bar"]
     check.check(instance)
-    assert len(aggregator.service_checks(SERVICE_CHECK)) == 1
-    sc = aggregator.service_checks(SERVICE_CHECK)[0]
+    assert len(aggregator.service_checks(common.SERVICE_CHECK)) == 1
+    sc = aggregator.service_checks(common.SERVICE_CHECK)[0]
+    assert sc.status == check.OK
+    assert sc.tags == tags
+
+
+def test_service_with_socket_ok(check, instance_socket, aggregator, memcached_socket):
```

Comment:
Let's put `@pytest.mark.skipif(platform.system() == 'Windows', reason='Windows sockets are not file handles')`